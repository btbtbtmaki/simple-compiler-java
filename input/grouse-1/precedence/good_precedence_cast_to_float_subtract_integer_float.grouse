main {

  print 80 :float - 7.9 nl;
  print 56 :float - 5.0 nl;
  print 56 :float - 4.1 nl;
  print 59 :float - 4.1 nl;
  print 97 :float - 8.3 nl;
  print 91 :float - 4.1 nl;
  print 20 :float - 7.5 nl;
  print 71 :float - 5.7 nl;
  print 77 :float - 3.3 nl;
  print 21 :float - 8.8 nl;
  print 91 :float - 5.8 nl;
  print 2 :float - 5.6 nl;

  print 2.4 - 47 :float nl;
  print 8.5 - 21 :float nl;
  print 8.4 - 48 :float nl;
  print 6.6 - 6 :float nl;
  print 5.4 - 83 :float nl;
  print 2.0 - 84 :float nl;
  print 7.9 - 68 :float nl;
  print 4.8 - 22 :float nl;
  print 5.0 - 4 :float nl;
  print 9.1 - 11 :float nl;
  print 1.2 - 32 :float nl;
  print 9.8 - 12 :float nl;

  print 62 :float - 16 :float nl;
  print 53 :float - 88 :float nl;
  print 82 :float - 86 :float nl;
  print 20 :float - 80 :float nl;
  print 96 :float - 1 :float nl;
  print 29 :float - 44 :float nl;
  print 63 :float - 53 :float nl;
  print 9 :float - 39 :float nl;
  print 30 :float - 39 :float nl;
  print 77 :float - 45 :float nl;
  print 85 :float - 63 :float nl;
  print 73 :float - 37 :float nl;

}

