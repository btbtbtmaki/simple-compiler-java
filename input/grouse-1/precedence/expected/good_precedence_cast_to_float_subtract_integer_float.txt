72.1
51
51.9
54.9
88.7
86.9
12.5
65.3
73.7
12.2
85.2
-3.6
-44.6
-12.5
-39.6
0.6
-77.6
-82
-60.1
-17.2
1
-1.9
-30.8
-2.2
46
-35
-4
-60
95
-15
10
-30
-9
32
22
36
